platform/javascript/display_server_javascript.h

```diff
@@ -139,7 +139,7 @@ class DisplayServerJavaScript : public DisplayServer {
 	virtual Rect2i screen_get_usable_rect(int p_screen = SCREEN_OF_MAIN_WINDOW) const override;
 	virtual int screen_get_dpi(int p_screen = SCREEN_OF_MAIN_WINDOW) const override;
 	virtual float screen_get_scale(int p_screen = SCREEN_OF_MAIN_WINDOW) const override;
-
+	virtual float screen_get_refresh_rate(int p_screen = SCREEN_OF_MAIN_WINDOW) const override;
```

Comment:
It would be better to keep the line between the "screen" methods and the "virtual_keyboard" ones.